projects/cdk/directives/pan/pan.service.ts

```diff
@@ -38,8 +38,8 @@ export class TuiPanService extends Observable<readonly [number, number]> {
                     ),
                     pairwise(),
                     map(([first, second]) => {
```

Comment:
It's pairwise, I thought it is typed as tuple with both element always there? We shouldn't be checking such cases.

Reply:
<img width="776" alt="image" src="https://github.com/user-attachments/assets/d1750fd0-10b0-4aee-97cb-2ab259cf8da4">

`touches[0]` return undefined

Reply:
Yes, but not `first` or `second`? I mean we do not need `?.` after those.

Reply:
<img width="725" alt="image" src="https://github.com/user-attachments/assets/d7a73b16-6b96-4f1c-bcd5-d35b654908ea">

Reply:
<img width="819" alt="image" src="https://github.com/user-attachments/assets/c998596b-4aa8-4aec-b687-4e57938121c6">